script/release/uploaders/upload.py

```diff
@@ -94,6 +94,17 @@ def main():
     debug_zip = os.path.join(OUT_DIR, DEBUG_NAME)
     shutil.copy2(os.path.join(OUT_DIR, 'debug.zip'), debug_zip)
     upload_electron(release, debug_zip, args)
+    # Upload libc++.zip for linux only
+    cxx_zip = os.path.join(OUT_DIR, 'libc++.zip')
+    upload_electron(release, cxx_zip, args)
```

Comment:
As mentioned above, we should do a rename here to include platform and architecture as the ia32 and x64 zip files are materially different